README.md

```diff
@@ -32,9 +32,10 @@ Usage:
   iexec [pod filter] [remote command(s)] [flags]
 
 Flags:
-  -c, --container string   Container to exec into (defaults to container[0]
+  -c, --container string   Container to search
   -h, --help               help for iexec
   -l, --log-level string   log level (trace|debug|info|warn|error|fatal|panic)
+  -x, --naked              Decolorize output
   -n, --namespace string   Namespace to search
   -v, --vimMode            Vim Mode enabled
```

Comment:
suggestion: change "vimMode" to "vim-mode" or "vimmode" to be consistent with the other descriptive options such as "log-level" and "namespace"

Reply:
fixed